<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterArea</class>
 <widget class="QWidget" name="FilterArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>31</y>
     <width>205</width>
     <height>335</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="luminositeText">
          <property name="text">
           <string>Luminosité</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="luminositeSlider">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="luminositeBox">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>198</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="saturationText">
          <property name="text">
           <string>Saturation</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSlider" name="saturationSlider">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="saturationNumber">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>198</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="blurText">
          <property name="text">
           <string>Flou</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSlider" name="blurSlider">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="blurBox">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>198</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>188</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="LumFroidesFilterButton">
        <property name="text">
         <string>Lumières froides </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="SummerFiltreButton">
        <property name="text">
         <string>Summer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="OmbresChaudesFilterButton">
        <property name="text">
         <string>Ombres chaudes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="NbFilterButton">
        <property name="text">
         <string>Noir&amp;Blanc</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>luminositeBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>luminositeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>luminositeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>luminositeBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blurSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blurSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blurBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationNumber</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationNumber</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
